CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# v1.4.0-M1 (Feb 2020)
+  * 1.4 Language features
+  * Klib versioning
+  * Commonizer support
+  * Experimental mimalloc allocator support (-Xallocator=mimalloc) to improve execution time performance
+
```

Comment:
* Debug improvements?
* https://github.com/JetBrains/kotlin-native/pull/3778
* https://github.com/JetBrains/kotlin-native/pull/3822 and https://github.com/JetBrains/kotlin-native/pull/3842
* https://github.com/JetBrains/kotlin-native/pull/3742